homeassistant/components/zwave_js/strings.json

```
@@ -38,26 +38,33 @@
     },
     "step": {
       "configure_addon_user": {
+        "data": {
+          "network_type": "Is your network new or does it already exist?",
+          "usb_path": "[%key:common::config_flow::data::usb_path%]"
+        },
+        "description": "Select your Z-Wave adapter",
+        "title": "Enter the Z-Wave add-on configuration"
+      },
+      "configure_security_keys": {
         "data": {
           "lr_s2_access_control_key": "Long Range S2 Access Control Key",
           "lr_s2_authenticated_key": "Long Range S2 Authenticated Key",
           "s0_legacy_key": "S0 Key (Legacy)",
           "s2_access_control_key": "S2 Access Control Key",
           "s2_authenticated_key": "S2 Authenticated Key",
-          "s2_unauthenticated_key": "S2 Unauthenticated Key",
-          "usb_path": "[%key:common::config_flow::data::usb_path%]"
+          "s2_unauthenticated_key": "S2 Unauthenticated Key"
         },
-        "description": "Select your Z-Wave adapter",
-        "title": "Enter the Z-Wave add-on configuration"
+        "description": "Enter the security keys for your existing Z-Wave network",
+        "title": "Security Keys"
```

This should be sentence-cased as we can translate this headline – contrary to the individual key names which should be kept in translations.

```suggestion
        "title": "Security keys"
```